webp_build/builder.py

```diff
@@ -42,10 +43,10 @@ def get_arch() -> str:
     raise RuntimeError('Unable to determine the compilation target architecture')
 
 
-def install_libwebp(arch: str) -> dict:
+def install_libwebp(arch: str) -> "dict[Any, Any]":
```

Comment:
I think that the most Python 3.8-friendly way of handling dict and list would be to use typing.Dict and typing.List